### PR TITLE
fix(agents): harden GitHub issue AgentRun intake

### PR DESCRIPTION
**Pull request by gregkonush**
## Summary

- Stops Froussard from copying full GitHub issue prompts and bodies into AgentRun parameters, keeping long content in implementation text instead.
- Makes successful AgentRun audit writes non-blocking so audit table timeouts do not abort otherwise valid run creation.
- Collapses AgentRun idempotency reservation into one SQL statement and increases Agents DB pool/query-timeout headroom for observed checkpoint stalls.

## Related Issues

Related to #11885 and #11886 intake failures. This does not close the Torghut implementation work described in those issues.

## Testing

- `bun run --filter froussard test -- src/services/agents.test.ts`
- `bun run --filter @proompteng/agents test -- src/server/v1/agent-runs.test.ts`
- `bun run --filter froussard typecheck`
- `bun run --filter @proompteng/agents tsc`
- `bunx oxfmt --check apps/froussard/src/services/agents.ts apps/froussard/src/services/agents.test.ts services/agents/src/server/v1/agent-run-submit.ts services/agents/src/server/v1/agent-runs.test.ts services/agents/src/server/primitives-store.ts`
- `bun run --filter froussard lint:oxlint -- src/services/agents.ts src/services/agents.test.ts` (passes with existing package-wide warnings)
- `bun run --filter @proompteng/agents lint:oxlint -- src/server/v1/agent-run-submit.ts src/server/v1/agent-runs.test.ts src/server/primitives-store.ts` (passes with existing package-wide warnings)
- `helm template agents charts/agents -n agents -f argocd/applications/agents/values.yaml >/tmp/agents-render.yaml`
- `bun run --filter froussard build`

## Screenshots (if applicable)

N/A.

## Breaking Changes

None.

## Checklist

- [x] Testing section documents the exact validation performed (or `N/A` with justification).
- [x] Screenshots and Breaking Changes sections are handled appropriately (removed or filled in).
- [x] Documentation, release notes, and follow-ups are updated or tracked.

### CODE REVIEW

File: services/agents/src/server/primitives-store.ts [hunk 1]
```diff
@@ -663,38 +666,49 @@ export const createPrimitivesStore = (options: PrimitivesStoreOptions = {}): Pri
...
+        ON CONFLICT (namespace, agent_name, idempotency_key) DO NOTHING
+        RETURNING *, TRUE AS created
+      ),
+      existing AS (
+        SELECT *, FALSE AS created
+        FROM agent_run_idempotency_keys
+        WHERE namespace = ${input.namespace}
+          AND agent_name = ${input.agentName}
+          AND idempotency_key = ${input.idempotencyKey}
+          AND NOT EXISTS (SELECT 1 FROM inserted)
```
[the remark this answers is not in the document]
Comment (gregkonush) [reply services/agents/src/server/primitives-store.ts:698]: Addressed in #11895 and completed in #11898. #11895 removed the single-statement CTE and restored the insert plus separate post-conflict lookup so a loser of ON CONFLICT does not depend on the original statement snapshot. #11898 added the retry path for the stale-reclaim edge case. Validation: @proompteng/agents primitives-store tests, agent-runs idempotency/audit tests, oxfmt check, @proompteng/agents tsc, and PR CI all passed before merge.
